grafana/provisioning/dashboards/tc-dashboard/tc-dashboard.json

```diff
@@ -333,7 +333,7 @@
               "type": "sum"
             }
           ],
-          "query": "name:jvm_gc_memory_* AND instance:$instance",
+          "query": "name:jvm_gc_memory_* AND instance.keyword:$instance",
```

Comment:
?

Reply:
This is automatically done by Grafana. I don't know why to keep this if it's generated but since it is a `modification` - let it be. It is not very good to ignore existing files in repo.